crates/services/executor/src/executor.rs

```diff
@@ -338,11 +339,8 @@ pub struct ExecutionOptions {
     /// The flag allows the usage of fake coins in the inputs of the transaction.
     /// When `false` the executor skips signature and UTXO existence checks.
     pub forbid_fake_coins: bool,
-    /// Print execution backtraces if transaction execution reverts.
-    ///
-    /// Deprecated field. Do nothing. This fields exists for serialization and
-    /// deserialization compatibility.
```

Comment:
@xgreenx Doesn't replacing this filed break things? Or require a hard fork upgrade or something?